addOns/pscanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html

```diff
@@ -34,40 +34,45 @@ <H2>Content Cacheability</H2>
 </ul>
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CacheableScanRule.java">CacheableScanRule.java</a>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10049/">10049</a>
```

Comment:
Should have a `br` before, otherwise this will be in the same line.

Reply:
For all of them